Sprint-3/3-mandatory-practice/implement/get-ordinal-number.js

```diff
@@ -1,5 +1,14 @@
 function getOrdinalNumber(num) {
- return "1st";
+if (num === 11 || num ===12 || num ===13) return num + "th";
+const lastDigit = num % 10;
+if (lastDigit === 1) return num + "st";
+else if (lastDigit === 2) return num + "nd";
+else if (lastDigit === 3) return num + "rd";
+else return num + "th";
 }
 
+
+
```

Comment:
Good though it's important to format your code correctly for legibility and some languages like Python require it. You can do this manually or else there is a 'Format Document' option in VS Code which is probably preferable.